sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
I kind of think it as  pause/proceed loop. I am fine with either we keep or dont keep interrupts in steps.

One benefit of it in canary phase: phase is in waiting state, and first two steps are also waiting state, rest of the steps are pending. When you say continue, only one of those steps will change to pending then in_progress, but second one will be waiting. So you know that you need to say continue one more time (0.8.1.2 DefaultStep returns waiting if interrupted and pending).